test/helpers.ts

```diff
@@ -261,8 +261,7 @@ export async function makeProtocol(opts: ProtocolOpts = {}): Promise<Protocol> {
     const initial = config.initial || 1e6;
     const name = config.name || symbol;
     const factory = config.factory || FaucetFactory;
-    let token;
-    token = (tokens[symbol] = await factory.deploy(initial, name, decimals, symbol));
```

Comment:
not exactly sure how we ended up with these strange lines, where we declare `token` and define it on the next line